avgerou/roster.json

```diff
@@ -20,7 +20,12 @@
     },
     {
       "name": "Tomislav Dinev",
-      "github": "dinevtomislav"
+      "github": "dinevtomislav",
+      "materials": [
+        {
+          "type2":"cv"
+        }
+      ]
```

Comment:
Can you please change the key, it should be type and not type2 😄 ?

Reply:
No problem! Should be done now.